README.md

```diff
@@ -245,40 +245,40 @@ We are looking for maintainers to write query files for their languages.
 
 List of currently supported languages:
 
-- [x] [bash](https://github.com/tree-sitter/tree-sitter-bash) (maintained by @TravonteD)
-- [x] [c](https://github.com/tree-sitter/tree-sitter-c) (maintained by @vigoux)
-- [x] [c_sharp](https://github.com/tree-sitter/tree-sitter-c-sharp) (maintained by @svermeulen)
-- [x] [cpp](https://github.com/tree-sitter/tree-sitter-cpp) (maintained by @theHamsta)
-- [x] [css](https://github.com/tree-sitter/tree-sitter-css) (maintained by @TravonteD)
-- [x] [dart](https://github.com/UserNobody14/tree-sitter-dart) (maintained by @Akin909)
-- [ ] [elm](https://github.com/razzeee/tree-sitter-elm)
-- [x] [go](https://github.com/tree-sitter/tree-sitter-go) (maintained by @theHamsta, @WinWisely268)
-- [ ] [haskell](https://github.com/tree-sitter/tree-sitter-haskell)
-- [ ] [html](https://github.com/tree-sitter/tree-sitter-html)
-- [x] [java](https://github.com/tree-sitter/tree-sitter-java) (maintained by p00f)
-- [x] [javascript](https://github.com/tree-sitter/tree-sitter-javascript) (maintained by @steelsojka)
-- [x] [jsdoc](https://github.com/tree-sitter/tree-sitter-jsdoc) (maintained by @steelsojka)
-- [x] [json](https://github.com/tree-sitter/tree-sitter-json) (maintained by @steelsojka)
-- [ ] [julia](https://github.com/tree-sitter/tree-sitter-julia)
-- [ ] [lisp](https://github.com/theHamsta/tree-sitter-clojure)
-- [x] [lua](https://github.com/nvim-treesitter/tree-sitter-lua) (maintained by @vigoux)
-- [ ] [markdown](https://github.com/ikatyang/tree-sitter-markdown)
-- [ ] [nix](https://github.com/cstrahan/tree-sitter-nix)
-- [ ] [ocaml](https://github.com/tree-sitter/tree-sitter-ocaml)
-- [x] [php](https://github.com/tree-sitter/tree-sitter-php) (maintained by @tk-shirasaka)
-- [x] [python](https://github.com/tree-sitter/tree-sitter-python) (maintained by @theHamsta)
-- [x] [Tree-sitter query language](https://github.com/nvim-treesitter/tree-sitter-query) (maintained by @steelsojka)
-- [x] [regex](https://github.com/tree-sitter/tree-sitter-regex) (maintained by @theHamsta)
-- [x] [rst](https://github.com/stsewd/tree-sitter-rst) (maintained by @stsewd)
-- [ ] [ruby](https://github.com/tree-sitter/tree-sitter-ruby)
-- [x] [rust](https://github.com/tree-sitter/tree-sitter-rust) (maintained by @vigoux)
-- [ ] [scala](https://github.com/tree-sitter/tree-sitter-scala)
-- [ ] [swift](https://github.com/tree-sitter/tree-sitter-swift)
-- [x] [toml](https://github.com/ikatyang/tree-sitter-toml) (maintained by @tk-shirasaka)
-- [ ] [tsx](https://github.com/tree-sitter/tree-sitter-typescript)
-- [x] [typescript](https://github.com/tree-sitter/tree-sitter-typescript) (maintained by @steelsojka)
-- [ ] [vue](https://github.com/ikatyang/tree-sitter-vue)
-- [ ] [yaml](https://github.com/ikatyang/tree-sitter-yaml)
+- [x] [bash](https://github.com/tree-sitter/tree-sitter-bash) (maintained by @TravonteD) [✅](queries/bash/highlights.scm "highlights")[✅](queries/bash/locals.scm "locals")[❌](queries/bash/textobjects.scm "textobjects")
```

Comment:
I think you can make this a sublist so is easy to read.

- [x] Language (maintainers)
  - [x] highlights
  - [ ] textobjects

Reply:
Don't know whether this is a good idea at all...
Would be only symbols when rendered https://github.com/theHamsta/nvim-treesitter/tree/parser-links-readme

Sublists would make it very long.

Reply:
Maybe a `<detail>`
<detail>
very long info
</detail>

Reply:
With icons:

- [x] [bash](https://github.com/tree-sitter/tree-sitter-bash) (maintained by @TravonteD) [✅](queries/bash/highlights.scm "highlights")[✅](queries/bash/locals.scm "locals")[❌](queries/bash/textobjects.scm "textobjects")
- [x] [c](https://github.com/tree-sitter/tree-sitter-c) (maintained by @vigoux) [✅](queries/c/highlights.scm "highlights")[✅](queries/c/locals.scm "locals")[✅](queries/c/textobjects.scm "textobjects")
- [x] [c_sharp](https://github.com/tree-sitter/tree-sitter-c-sharp) (maintained by @svermeulen) [✅](queries/c_sharp/highlights.scm "highlights")[❌](queries/c_sharp/locals.scm "locals")[✅](queries/c_sharp/textobjects.scm "textobjects")
- [x] [cpp](https://github.com/tree-sitter/tree-sitter-cpp) (maintained by @theHamsta) [✅](queries/cpp/highlights.scm "highlights")[✅](queries/cpp/locals.scm "locals")[✅](queries/cpp/textobjects.scm "textobjects")
- [x] [css](https://github.com/tree-sitter/tree-sitter-css) (maintained by @TravonteD) [✅](queries/css/highlights.scm "highlights")[❌](queries/css/locals.scm "locals")[❌](queries/css/textobjects.scm "textobjects")
- [x] [dart](https://github.com/UserNobody14/tree-sitter-dart) (maintained by @Akin909) [✅](queries/dart/highlights.scm "highlights")[❌](queries/dart/locals.scm "locals")[❌](queries/dart/textobjects.scm "textobjects")
- [ ] [elm](https://github.com/razzeee/tree-sitter-elm) [❌](queries/elm/highlights.scm "highlights")[❌](queries/elm/locals.scm "locals")[❌](queries/elm/textobjects.scm "textobjects")
- [x] [go](https://github.com/tree-sitter/tree-sitter-go) (maintained by @theHamsta, @WinWisely268) [✅](queries/go/highlights.scm "highlights")[✅](queries/go/locals.scm "locals")[✅](queries/go/textobjects.scm "textobjects")
- [ ] [haskell](https://github.com/tree-sitter/tree-sitter-haskell) [❌](queries/haskell/highlights.scm "highlights")[❌](queries/haskell/locals.scm "locals")[❌](queries/haskell/textobjects.scm "textobjects")
- [ ] [html](https://github.com/tree-sitter/tree-sitter-html) [✅](queries/html/highlights.scm "highlights")[✅](queries/html/locals.scm "locals")[❌](queries/html/textobjects.scm "textobjects")
- [x] [java](https://github.com/tree-sitter/tree-sitter-java) (maintained by p00f) [✅](queries/java/highlights.scm "highlights")[✅](queries/java/locals.scm "locals")[❌](queries/java/textobjects.scm "textobjects")
- [x] [javascript](https://github.com/tree-sitter/tree-sitter-javascript) (maintained by @steelsojka) [✅](queries/javascript/highlights.scm "highlights")[✅](queries/javascript/locals.scm "locals")[✅](queries/javascript/textobjects.scm "textobjects")
- [x] [jsdoc](https://github.com/tree-sitter/tree-sitter-jsdoc) (maintained by @steelsojka) [❌](queries/jsdoc/highlights.scm "highlights")[❌](queries/jsdoc/locals.scm "locals")[❌](queries/jsdoc/textobjects.scm "textobjects")
- [x] [json](https://github.com/tree-sitter/tree-sitter-json) (maintained by @steelsojka) [✅](queries/json/highlights.scm "highlights")[❌](queries/json/locals.scm "locals")[❌](queries/json/textobjects.scm "textobjects")
- [ ] [julia](https://github.com/tree-sitter/tree-sitter-julia) [❌](queries/julia/highlights.scm "highlights")[❌](queries/julia/locals.scm "locals")[❌](queries/julia/textobjects.scm "textobjects")
- [ ] [lisp](https://github.com/theHamsta/tree-sitter-clojure) [✅](queries/lisp/highlights.scm "highlights")[❌](queries/lisp/locals.scm "locals")[❌](queries/lisp/textobjects.scm "textobjects")
- [x] [lua](https://github.com/nvim-treesitter/tree-sitter-lua) (maintained by @vigoux) [✅](queries/lua/highlights.scm "highlights")[✅](queries/lua/locals.scm "locals")[✅](queries/lua/textobjects.scm "textobjects")
- [ ] [markdown](https://github.com/ikatyang/tree-sitter-markdown) [❌](queries/markdown/highlights.scm "highlights")[❌](queries/markdown/locals.scm "locals")[❌](queries/markdown/textobjects.scm "textobjects")
- [ ] [nix](https://github.com/cstrahan/tree-sitter-nix) [❌](queries/nix/highlights.scm "highlights")[❌](queries/nix/locals.scm "locals")[❌](queries/nix/textobjects.scm "textobjects")
- [ ] [ocaml](https://github.com/tree-sitter/tree-sitter-ocaml) [❌](queries/ocaml/highlights.scm "highlights")[❌](queries/ocaml/locals.scm "locals")[❌](queries/ocaml/textobjects.scm "textobjects")
- [x] [php](https://github.com/tree-sitter/tree-sitter-php) (maintained by @tk-shirasaka) [✅](queries/php/highlights.scm "highlights")[✅](queries/php/locals.scm "locals")[❌](queries/php/textobjects.scm "textobjects")
- [x] [python](https://github.com/tree-sitter/tree-sitter-python) (maintained by @theHamsta) [✅](queries/python/highlights.scm "highlights")[✅](queries/python/locals.scm "locals")[✅](queries/python/textobjects.scm "textobjects")
- [x] [Tree-sitter query language](https://github.com/nvim-treesitter/tree-sitter-query) (maintained by @steelsojka) [✅](queries/query/highlights.scm "highlights")[✅](queries/query/locals.scm "locals")[❌](queries/query/textobjects.scm "textobjects")
- [x] [regex](https://github.com/tree-sitter/tree-sitter-regex) (maintained by @theHamsta) [✅](queries/regex/highlights.scm "highlights")[❌](queries/regex/locals.scm "locals")[❌](queries/regex/textobjects.scm "textobjects")
- [x] [rst](https://github.com/stsewd/tree-sitter-rst) (maintained by @stsewd) [✅](queries/rst/highlights.scm "highlights")[✅](queries/rst/locals.scm "locals")[✅](queries/rst/textobjects.scm "textobjects")
- [ ] [ruby](https://github.com/tree-sitter/tree-sitter-ruby) [✅](queries/ruby/highlights.scm "highlights")[✅](queries/ruby/locals.scm "locals")[❌](queries/ruby/textobjects.scm "textobjects")
- [x] [rust](https://github.com/tree-sitter/tree-sitter-rust) (maintained by @vigoux) [✅](queries/rust/highlights.scm "highlights")[✅](queries/rust/locals.scm "locals")[✅](queries/rust/textobjects.scm "textobjects")
- [ ] [scala](https://github.com/tree-sitter/tree-sitter-scala) [❌](queries/scala/highlights.scm "highlights")[❌](queries/scala/locals.scm "locals")[❌](queries/scala/textobjects.scm "textobjects")
- [ ] [swift](https://github.com/tree-sitter/tree-sitter-swift) [❌](queries/swift/highlights.scm "highlights")[❌](queries/swift/locals.scm "locals")[❌](queries/swift/textobjects.scm "textobjects")
- [x] [toml](https://github.com/ikatyang/tree-sitter-toml) (maintained by @tk-shirasaka) [✅](queries/toml/highlights.scm "highlights")[✅](queries/toml/locals.scm "locals")[❌](queries/toml/textobjects.scm "textobjects")
- [ ] [tsx](https://github.com/tree-sitter/tree-sitter-typescript) [✅](queries/tsx/highlights.scm "highlights")[✅](queries/tsx/locals.scm "locals")[✅](queries/tsx/textobjects.scm "textobjects")
- [x] [typescript](https://github.com/tree-sitter/tree-sitter-typescript) (maintained by @steelsojka) [✅](queries/typescript/highlights.scm "highlights")[✅](queries/typescript/locals.scm "locals")[✅](queries/typescript/textobjects.scm "textobjects")
- [ ] [vue](https://github.com/ikatyang/tree-sitter-vue) [❌](queries/vue/highlights.scm "highlights")[❌](queries/vue/locals.scm "locals")[❌](queries/vue/textobjects.scm "textobjects")
- [ ] [yaml](https://github.com/ikatyang/tree-sitter-yaml) [❌](queries/yaml/highlights.scm "highlights")[❌](queries/yaml/locals.scm "locals")[❌](queries/yaml/textobjects.scm "textobjects")

Without:
- [x] [bash](https://github.com/tree-sitter/tree-sitter-bash) (maintained by @TravonteD)
- [x] [c](https://github.com/tree-sitter/tree-sitter-c) (maintained by @vigoux)
- [x] [c_sharp](https://github.com/tree-sitter/tree-sitter-c-sharp) (maintained by @svermeulen)
- [x] [cpp](https://github.com/tree-sitter/tree-sitter-cpp) (maintained by @theHamsta)
- [x] [css](https://github.com/tree-sitter/tree-sitter-css) (maintained by @TravonteD)
- [x] [dart](https://github.com/UserNobody14/tree-sitter-dart) (maintained by @Akin909)
- [ ] [elm](https://github.com/razzeee/tree-sitter-elm)
- [x] [go](https://github.com/tree-sitter/tree-sitter-go) (maintained by @theHamsta, @WinWisely268)
- [ ] [haskell](https://github.com/tree-sitter/tree-sitter-haskell)
- [ ] [html](https://github.com/tree-sitter/tree-sitter-html)
- [x] [java](https://github.com/tree-sitter/tree-sitter-java) (maintained by p00f)
- [x] [javascript](https://github.com/tree-sitter/tree-sitter-javascript) (maintained by @steelsojka)
- [x] [jsdoc](https://github.com/tree-sitter/tree-sitter-jsdoc) (maintained by @steelsojka)
- [x] [json](https://github.com/tree-sitter/tree-sitter-json) (maintained by @steelsojka)
- [ ] [julia](https://github.com/tree-sitter/tree-sitter-julia)
- [ ] [lisp](https://github.com/theHamsta/tree-sitter-clojure)
- [x] [lua](https://github.com/nvim-treesitter/tree-sitter-lua) (maintained by @vigoux)
- [ ] [markdown](https://github.com/ikatyang/tree-sitter-markdown)
- [ ] [nix](https://github.com/cstrahan/tree-sitter-nix)
- [ ] [ocaml](https://github.com/tree-sitter/tree-sitter-ocaml)
- [x] [php](https://github.com/tree-sitter/tree-sitter-php) (maintained by @tk-shirasaka)
- [x] [python](https://github.com/tree-sitter/tree-sitter-python) (maintained by @theHamsta)
- [x] [Tree-sitter query language](https://github.com/nvim-treesitter/tree-sitter-query) (maintained by @steelsojka)
- [x] [regex](https://github.com/tree-sitter/tree-sitter-regex) (maintained by @theHamsta)
- [x] [rst](https://github.com/stsewd/tree-sitter-rst) (maintained by @stsewd)
- [ ] [ruby](https://github.com/tree-sitter/tree-sitter-ruby)
- [x] [rust](https://github.com/tree-sitter/tree-sitter-rust) (maintained by @vigoux)
- [ ] [scala](https://github.com/tree-sitter/tree-sitter-scala)
- [ ] [swift](https://github.com/tree-sitter/tree-sitter-swift)
- [x] [toml](https://github.com/ikatyang/tree-sitter-toml) (maintained by @tk-shirasaka)
- [ ] [tsx](https://github.com/tree-sitter/tree-sitter-typescript)
- [x] [typescript](https://github.com/tree-sitter/tree-sitter-typescript) (maintained by @steelsojka)
- [ ] [vue](https://github.com/ikatyang/tree-sitter-vue)
- [ ] [yaml](https://github.com/ikatyang/tree-sitter-yaml)

Reply:
Also an argument for not including parser status: contributors that add a new query file need to run the script or else CI will complain that README is not up-to-date.

I guess *parser, link, maintainer* is enough.